Kongres.Api/Kongres.Api.WebApi/appsettings.json

```diff
@@ -18,5 +18,10 @@
     "Issuer": "localhost:5001",
     "Audience": "localhost:5001",
     "MinutesToExpire": 5
+  },
+  "Email": {
+    "Server": "127.0.0.1",
+    "SenderName": "Scienture",
+    "SenderEmail": "noreply@Scienture.com"
```

Comment:
please change name to "Scienture conference" and mail to "noreply@scientureconf.com"